### PR TITLE
Update required APIs for testing Looker TF samples

### PR DESCRIPTION
**Pull request by ronnieplooker**
## Description

Relates to [PR 479](https://github.com/terraform-google-modules/terraform-docs-samples/pull/479); no separate Buganizer ticket.

### CODE REVIEW

**Comment by msampathkumar** [comment]
/gcbrun

Invoking CICD tests

File: google_looker_instance/looker_instance_enterprise_full/main.tf [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START looker_google_looker_instance_enterprise_full]
+resource "google_looker_instance" "looker-instance" {
```
Comment (msampathkumar) [inline google_looker_instance/looker_instance_enterprise_full/main.tf:18]:
```suggestion
resource "google_looker_instance" "default" {
```

Please use `default` as the resource name, unless you have name conflict issues.

File: google_looker_instance/looker_instance_enterprise_full/main.tf [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  oauth_config {
+    client_id     = "my-client-id"
+    client_secret = "my-client-secret"
+  }
+  depends_on = [
+    google_service_networking_connection.looker_vpc_connection
+  ]
+}
+
+resource "google_service_networking_connection" "looker_vpc_connection" {
```
Comment (msampathkumar) [inline google_looker_instance/looker_instance_enterprise_full/main.tf:68]: Please use `default` as the resource name, unless you have name conflict issues.
```suggestion
resource "google_service_networking_connection" "default" {
```

File: google_looker_instance/looker_instance_enterprise_full/main.tf [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  network                 = data.google_compute_network.looker_network.id
+  service                 = "servicenetworking.googleapis.com"
+  reserved_peering_ranges = [google_compute_global_address.looker_range.name]
+}
+
+resource "google_compute_global_address" "looker_range" {
```
Comment (msampathkumar) [inline google_looker_instance/looker_instance_enterprise_full/main.tf:74]: Please use `default` as the resource name, unless you have name conflict issues.
```suggestion
resource "google_compute_global_address" "default" {
```

File: google_looker_instance/looker_instance_enterprise_full/main.tf [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  name          = "looker-range"
+  purpose       = "VPC_PEERING"
+  address_type  = "INTERNAL"
+  prefix_length = 20
+  network       = data.google_compute_network.looker_network.id
+}
+
+data "google_project" "project" {}
```
Comment (msampathkumar) [inline google_looker_instance/looker_instance_enterprise_full/main.tf:82]:
```suggestion
data "google_project" "default" {}
```

File: google_looker_instance/looker_instance_enterprise_full/main.tf [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+data "google_compute_network" "looker_network" {
+  name = "looker-network"
+}
+
+resource "google_kms_crypto_key_iam_member" "crypto_key" {
+  crypto_key_id = "looker-kms-key"
+  role          = "roles/cloudkms.cryptoKeyEncrypterDecrypter"
+  member        = "serviceAccount:service-${data.google_project.project.number}@gcp-sa-looker.iam.gserviceaccount.com"
```
Comment (msampathkumar) [inline google_looker_instance/looker_instance_enterprise_full/main.tf:91]:
```suggestion
  member        = "serviceAccount:service-${data.google_project.default.number}@gcp-sa-looker.iam.gserviceaccount.com"
```

File: google_looker_instance/looker_instance_enterprise_full/main.tf [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+  network       = data.google_compute_network.looker_network.id
+}
+
+data "google_project" "project" {}
+
+data "google_compute_network" "looker_network" {
+  name = "looker-network"
+}
+
+resource "google_kms_crypto_key_iam_member" "crypto_key" {
```
Comment (msampathkumar) [inline google_looker_instance/looker_instance_enterprise_full/main.tf:88]:
```suggestion
resource "google_kms_crypto_key_iam_member" "default" {
```

**Comment by rogerthatdev** [comment]
Closing this PR in lieu of #479 which includes these changes

**Pull request closed without merging**